### ISSUE TITLE
Individual ID mismatch between case individuals and variants individuals

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#2721]
This bug is causing the page to crash:

```
2021-06-23 13:04:30,034 - scout.server.app - ERROR: Exception on /cust003/21115/1ea25a766a24eb0fdcf8d8fba576346e [GET] [in /home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py:1892]
Traceback (most recent call last):
  File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/hiseq.clinical/SERVER/apps/scout/git/scout/scout/server/utils.py", line 26, in decorated_function
    context = func(*args, **kwargs)
  File "/home/hiseq.clinical/SERVER/apps/scout/git/scout/scout/server/blueprints/variant/views.py", line 44, in variant
    data = variant_controller(store, institute_id, case_name, variant_id=variant_id)
  File "/home/hiseq.clinical/SERVER/apps/scout/git/scout/scout/server/blueprints/variant/controllers.py", line 226, in variant
    is_affected(variant_obj, case_obj)
  File "/home/hiseq.clinical/SERVER/apps/scout/git/scout/scout/server/blueprints/variant/utils.py", line 369, in is_affected
    individual = individuals[sample_obj.get("sample_id")]
KeyError: 'ACC7929A1'
```

**Comment by northwestwitch** [comment]
It's caused by individual ID that is different between case:

![image](https://user-images.githubusercontent.com/28093618/123089067-c2624d00-d426-11eb-81ef-e00c60c6e635.png)


And variant object:

![image](https://user-images.githubusercontent.com/28093618/123089329-166d3180-d427-11eb-8980-8a65c84616e7.png)

Could be one of those cases in #2702?

**Comment by northwestwitch** [comment]
Checking in statusDB I've found that there is another family (exactcicada) with the same customer (cust003), same scout display name (21115) and an individual with ID ACC7929A1 instead (but same individual display name). This case was analysed 12 days later than the other (2021-05-22, see https://clinical-api.scilifelab.se/admin/sample/?search=ACC7929A1). It says it was uploaded. I'm a bit confused now. Perhaps we have messed up and uploaded the same case twice, with a different internal ID but same display name and same display name of the case individual?

**Comment by dnil** [comment]
Yes, at any rate we need to check against that on load when we later use the combination to identify the cases uniquely in queries. In general I am not so comfortable with multiple cases with the same display name from the same institute, even if we were to change all case lookups to only use internal id. Would be a bit difficult for the users to keep track of them?!

**Comment by northwestwitch** [comment]
I'm not even sure it's the customer that ordered an analysis of 2 cases with same name in this case. Perhaps we rerunned the case for some reason and just uploaded the second one on top of the first

**Comment by northwestwitch** [comment]
This should be fixed now, after merging #2727
